test/client/fixtures/transport-test.js

```diff
@@ -81,7 +81,7 @@ asyncTest('queuedAsyncServiceMsg', function () {
             var expectedCompleteMsgReqs = [10, 500, 200, 300, 200];
 
             deepEqual(completeMsgReqs, expectedCompleteMsgReqs);
-            Transport.asyncServiceMsg = savedAsyncServiceMsgFunc;
+            Transport.asyncServiceMsg   = savedAsyncServiceMsgFunc;
```

Comment:
Unnecessary extra space. Seems like it's formatting bug. Just add a newline to avoid it (actually it's always better to separate assertions from the other code blocks as well as variable assignments)